src/miner.cpp

```diff
@@ -269,6 +273,7 @@ int BlockAssembler::UpdatePackagesForAdded(const CTxMemPool::setEntries& already
 bool BlockAssembler::SkipMapTxEntry(CTxMemPool::txiter it, indexed_modified_transaction_set &mapModifiedTx, CTxMemPool::setEntries &failedTx)
 {
     assert (it != mempool.mapTx.end());
+    if (it->GetTime() > m_max_tx_time) return true; // txn too recent
```

Comment:
I suggested rename eslewhere to be self-describing(something like `m_skip_inclusion_until` ?) and then you can drop the comment on this line because it's obvious what the meaning is.

Reply:
updated name to `m_skip_inclusion_until` & added documentation to this function definition & in the header